packages/obojobo-document-engine/__tests__/server/attempt-start.test.js

```diff
@@ -240,32 +253,32 @@ describe('start attempt route', () => {
 		expect(usedQuestionMap.get('qb2.q2')).toBe(0)
 	})
 
-	test('initAssessmentUsedQuestions tracks question use from initalized map', () => {
-		const fakeChildNodes = [
-			{
-				id: 'qb1.q1',
-				children: []
-			},
-			{
-				id: 'qb1.q2',
-				children: []
-			},
-			{
-				id: 'mockId',
-				children: []
-			}
-		]
-		const mockQbTree = { id: 'qb1', children: fakeChildNodes }
-
-		initAssessmentUsedQuestions(mockQbTree, mockUsedQuestionMap)
-
-		expect(mockUsedQuestionMap.get('qb1')).toBe(1)
-		expect(mockUsedQuestionMap.get('qb1.q1')).toBe(1)
-		expect(mockUsedQuestionMap.get('qb1.q2')).toBe(1)
-		expect(mockUsedQuestionMap.get('qb2')).toBe(0)
-		expect(mockUsedQuestionMap.get('qb2.q1')).toBe(0)
-		expect(mockUsedQuestionMap.get('qb2.q2')).toBe(0)
-	})
+	// test('initAssessmentUsedQuestions tracks question use from initalized map', () => {
```

Comment:
Removed these tests, and will delete before merge. Want to make sure the functions they cover are 100% safe to delete.